docs/experience/weekly-health-reports.md

```diff
@@ -0,0 +1,30 @@
+title: Weekly Health Reports
+description: Sematext Experience weekly health reports are automatic emails sent each week to provide a snapshot of your website's performance metrics and user interactions.
+
+Weekly Health reports are automatically sent on Monday and provide a snapshot of your website’s performance and user interactions over the past week. The email compiles different metrics compared to the previous week and the last four weeks, giving you a quick glance at the information and helping you decide if further investigation is necessary without opening Sematext Cloud.
+
+## What's in the Weekly Health Report
+
+To configure sampling for your App:
+
+**Page Loads**: Compares how many times users loaded pages on your website during the week and rates the performance with predefined [Apdex scores](https://sematext.com/docs/experience/configure-requirements/) as excellent, good, fair or poor. It also compares the current status with the previous week and the last four weeks and highlights differences.
+
+![Sematext Cloud Logs Guide - Connect Apps](../images/experience/page-views.png)
```

Comment:
"Sematext Cloud Logs Guide - Connect Apps" looks erroneous.  Same with 3 more instances of that below.